plugins/utils/video_chat/__main__.py

```diff
@@ -948,27 +1107,34 @@ async def play_video(file: str, height: int, width: int, quality: int):
                     r_width,
                     r_height,
                     25
-                )
+                ),
+                additional_ffmpeg_parameters=ffmpeg_parm
             )
         )
+    if not seek:
+        CURRENT_SONG['start'] = time.time()
 
 
-async def play_audio(file: str):
+async def play_audio(file: str, seek: int = None):
+    ffmpeg_parm = f'-ss {seek} -atend -to {CURRENT_SONG["duration"]}' if seek else ''
     try:
         await call.change_stream(
             CHAT_ID,
             AudioPiped(
-                file
+                file,
+                additional_ffmpeg_parameters=ffmpeg_parm
             )
         )
     except NotInGroupCallError:
         await call.join_group_call(
             CHAT_ID,
             AudioPiped(
-                file
+                file,
+                additional_ffmpeg_parameters=ffmpeg_parm
             ),
             stream_type=StreamType().pulse_stream,
         )
+    CURRENT_SONG['start'] = time.time()
```

Comment:
@subinps seems you need to add same check for seeking that you imposed [here](https://github.com/UsergeTeam/Userge-Plugins/blob/61ec4ce09da2d25b9bd81ed9d5f9017b88c70c85/plugins/utils/video_chat/__main__.py#L1114) , while seeking audio files

Reply:
thanks :)